pom.xml

```diff
@@ -59,10 +59,17 @@
 		    <artifactId>gson</artifactId>
 		    <version>2.8.2</version>
 		</dependency>
-		<!-- https://mvnrepository.com/artifact/com.fasterxml.jackson.core/jackson-core -->
-	
-		
-		</dependencies>
+		<dependency>
```

Comment:
commit proper formatted code, do for all files. import eclipse format code that I sent you.